### DISCUSSION TITLE
What are the values in the VAR/0.0.0 key?

### DISCUSSION THREAD

**Discussion by raybellwaves** [fsspec/kerchunk#422] [Q&A] [answered]
Going through https://projectpythia.org/kerchunk-cookbook/notebooks/foundations/02_kerchunk_multi_file.html.

When looking at h5chunks_translated['refs']["ACSNOW/0.0.0"] the output looks like:

['s3://wrf-se-ak-ar5/ccsm/rcp85/daily/2060/WRFDS_2060-01-01.nc', 3133401, 7839]

I'm curious what these values are. The first is the file path, the second byte start? and third byte range?

I wanted to know what are the differences between the translation of s3://wrf-se-ak-ar5/ccsm/rcp85/daily/2060/WRFDS_2060-01-01.nc and s3://wrf-se-ak-ar5/ccsm/rcp85/daily/2060/WRFDS_2060-01-02.nc. I thought with time being a day ahead there would be a lot of duplication of variables. But it looks like you can't encode one file and use that as a template for another file. I guess it depends on field values i.e. if a variables has lots of NaNs in one time step and float in another time step then the byte start and byte range will be different for the same field in different files.

**Comment by martindurant** [comment, marked as answer]
The [spec](https://fsspec.github.io/kerchunk/spec.html#version-1) says that the elements are
- URL
- byte offset
- byte size

Exactly where a given chunk ends up when writing the .nc depends on ... lots of things! Yes, there may be data-dependent factors, but also software version and environment. Honestly I don't know, which is why we always scan every file.
